test/builder/lib/ReplayableHelper.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.27;
+
+library ReplayableHelper {
+    function generateNonceFromSecret(bytes32 secret, uint256 replayCount) internal pure returns (bytes32) {
```

Comment:
Any reason not to use the optimized version above?

Reply:
Mainly just because we don't need the optimized version here to save us from stack too deep. Also I thought it's kinda nice to have a more readable version so we can sanity check the optimized version in our tests with a different implementation.